src/storage/store/table_statistics.cpp

```diff
@@ -13,7 +13,11 @@ TablesStatistics::TablesStatistics() {
 }
 
 void TablesStatistics::readFromFile(const std::string& directory) {
-    auto filePath = getTableStatisticsFilePath(directory, DBFileType::ORIGINAL);
```

Comment:
Maybe u can remove this api, and give dbFileType a default parameter
void TablesStatistics::readFromFile(const std::string& directory, common::DBFileType dbFileType = DBFileType::ORIGINAL)